account_move_print/report/journal_report.xml

```diff
@@ -0,0 +1,197 @@
+                                                />
+                                            </td>
+                                            <td name='tb_originator_tax'>
+                                                <span t-esc="line.tax_line_id.name" />
+                                            </td>
+                                            <td name='tb_taxes'>
+                                                <tr t-foreach="line.tax_ids" t-as="tax">
+                                                    <td>
+                                                       <span t-field="tax.name" />,
+                                                   </td>
+                                                </tr>
+                                            </td>
+                                        </tr>
+                                    </t>
+                                </tbody>
+                                <tfooter>
+                                    <tr>
+                                        <td name='tf_account' />
+                                        <td name='tf_partner' />
+                                        <td name='tf_line_name' />
+                                        <td
+                                            name='tf_analytic'
+                                            groups="analytic.group_analytic_accounting"
+                                        />
+                                        <td
+                                            name='tf_amount_currency'
+                                            groups="base.group_multi_currency"
+                                        />
+                                        <td name='tf_debit'>
+                                            <span
+                                                t-esc="total_debit"
+                                                t-options='{"widget": "monetary", "display_currency": o.company_currency_id}'
+                                            />
+                                        </td>
+                                        <td name='tf_credit'>
+                                            <span
+                                                t-esc="total_credit"
+                                                t-options='{"widget": "monetary", "display_currency": o.company_currency_id}'
+                                            />
+                                        </td>
+                                        <td name='tf_originator_tax' />
+                                        <td name='tb_taxes' />
+                                    </tr>
+                                </tfooter>
+                            </table>
+                        </div>
+                    </div>
+                </t>
+            </t>
+        </t>
+    </template>
+    <report
```

Comment:
This should change for new standard of reports creation

Reply:
@ChrisOForgeFlow suggestion done